src/device/mod.rs

```diff
@@ -852,6 +852,8 @@ fn check_auth (hh: &handshake::HalfHandshake, d: &mut LockReadGuard<Device>) {
                         .arg("wg")
```

Comment:
```suggestion
                        .arg("wg")
                        .arg("up")
```

So we'll actually need this as well.

Reply:
Might as well get this in in the same PR.

Reply:
Nice catch. This might explain why wireguard hasn't been functional lately on Freshscribe